### PR TITLE
 Adds option to pick repository to create changelog

### PR DESCRIPTION
**Pull request by camilasan**
- Pass repository name as the first parameter to php index.php generate:changelog command.
- Based on the repo it sets the default values for fetching the commits.
- Remove android.php
- Updates README

### CODE REVIEW

File: changelog/index.php [hunk 1]
```diff
@@ -37,28 +39,27 @@ protected function configure()
...
+                "files_videoplayer",
+                "firstrunwizard",
+                "gallery",
+                "logreader",
+                "nextcloud_announcements",
+                "notifications",
+                "password_policy",
+                "serverinfo",
+                "survey_client",
+            ];
```
Comment (MorrisJobke) [inline changelog/index.php]: Could you check that you used tabs with 4 spaces width? because this looks 🐟y.

File: changelog/index.php [hunk 2]
```diff
@@ -75,15 +76,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
 
 		$milestoneToCheck = null;
-		if (substr($base, 0, 1) === 'v') {
-			$version = explode('.', substr($base, 1));
+		$substring = 'v';
+		$subStringNum = 1;
+		if(strcasecmp($repoName, $server) != 0 ){
+            $reposToIterate = [$repoName];
+            $substring = 'stable-';
+            $subStringNum = 7;
```
Comment (MorrisJobke) [inline changelog/index.php]: Same here.

**Comment by MorrisJobke** [comment]
Thanks 👍 Unfortunately @tobiasKaminsky doesn't want to use that anymore :( But I guess you want to use it for the desktop repo later at some time, right?

File: changelog/index.php (hunk 1 above)
Comment (camilasan) [reply changelog/index.php]: Sure. Please, check again.

File: changelog/index.php (hunk 2 above)
Comment (camilasan) [reply changelog/index.php]: Done.

**Comment by camilasan** [comment]
@MorrisJobke I just used it, that is why I changed it. I found it very useful. :heart:

File: changelog/index.php [hunk 3]
```diff
@@ -37,10 +39,10 @@ protected function configure()
 	 */
 	protected function execute(InputInterface $input, OutputInterface $output)
 	{
-		// TODO iterate over all repos
-		$orgName = 'nextcloud';
-		$repoName = 'server';
+        $server = 'server';
+        $orgName = 'nextcloud';
```
Comment (MorrisJobke) [inline changelog/index.php]: Something is still wrong here.

File: changelog/index.php [hunk 4]
```diff
@@ -75,15 +76,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+		$output->writeln("repo: $repoName");
 		$output->writeln("base: $base");
 		$output->writeln("head: $head");
 
 		$milestoneToCheck = null;
-		if (substr($base, 0, 1) === 'v') {
-			$version = explode('.', substr($base, 1));
+		$substring = 'v';
+		$subStringNum = 1;
+		if(strcasecmp($repoName, $server) != 0 ){
```
Comment (MorrisJobke) [inline changelog/index.php]:
```suggestion
		if($repoName !== $server){
```

This should be enough. Even if this is not case sensitive.

Comment (camilasan) [reply changelog/index.php]: Done.

File: changelog/index.php (hunk 3 above)
Comment (camilasan) [reply changelog/index.php]: Sorry, I think it is correct now.

**Pull request merged**